api/src/pcapi/routes/serialization/public_api_collective_offers_serialize.py

```diff
@@ -60,6 +60,8 @@ def validate_email(email: str) -> str:
 
 
 def validate_emails(emails: list[str]) -> list[str]:
+    if len(emails) < 1:
```

Comment:
peut être remplacé par un `if not email`

Reply:
du coup je me rend compte qu'il y a deja une fonction :
```
def validate_email(email: str) -> str:
    if not email:
        raise ValueError("Ce champ ne peut pas être vide")
    if not email_utils.is_valid_email(email):
        raise ValueError(f"{email} n'est pas une adresse mail valide")
    return email

```